src/main/io/usb_cdc_hid.c

```diff
@@ -76,3 +84,12 @@ void sendRcDataToHid(void)
 #endif
 }
 #endif
+
+bool cdcDeviceIsMayBeActive()
+{
+#ifdef USE_USB_CDC_HID
+    return getBatteryState() == BATTERY_NOT_PRESENT && batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE && usbDevConfig()->type == COMPOSITE;
```

Comment:
Yes, I remove it because of the conversation above. There I followed your suggestions.
But still, I think that it is a better approach to check for `usbDevConfig()->type == COMPOSITE && usbVcpIsConnected()` than rely on a battery and a voltage sensor, where changing settings of the voltage sensor can screw this all thing up. So with `usbDevConfig()->type == COMPOSITE && usbVcpIsConnected()` we can cover up few more targets than with any other combination where `getBatteryState() == BATTERY_NOT_PRESENT && batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE` is in. If user has Sparky2 or similar board, with no V/C sensor present, which is by default and voltage meter in that case is NONE, that function will not work anyway.

Since we have `usbVcpIsConnected()` invoked here, there is no reason to rely on the battery state just to check the USB connection. So I can rewrite this function on this way:
```
bool cdcDeviceIsMayBeActive()
{
    return usbIsConnected() && usbDevConfig()->type == COMPOSITE;
}
```
Which will do the job better than checking a battery and sensors. 

Oh, btw, `usbVcpIsConnected()` is function dependent of the CPU architecture and it's better case to call `usbIsConnected()` here because some of the targets will fail in build.

Reply:
No, the idea is definitely that we want to allow users to invoke the CMS if they have a battery connected, even if USB is plugged in - `usbIsConnected()` is just an additonal assurance. 

So it should be:

`usbDevConfig()->type == COMPOSITE && usbIsConnected() && (getBatteryState() == BATTERY_NOT_PRESENT || batteryConfig()->voltageMeterSource == VOLTAGE_METER_NONE)'

Reply:
Ok, I agree with this.
I'll do the final code changes like you suggested.
Thanks.